os/arch/arm/src/common/up_internal.h

```diff
@@ -332,6 +413,60 @@ EXTERN uint32_t _eramfuncs;		/* Copy destination end address in RAM */
 #define CONFIG_MPU_STACK_GUARD_SIZE 0
 #endif
 
+#if defined(CONFIG_ARCH_CORTEXA32)
+void modifyreg8(unsigned int addr, uint8_t clearbits, uint8_t setbits);
+void modifyreg16(unsigned int addr, uint16_t clearbits, uint16_t setbits);
+void modifyreg32(unsigned int addr, uint32_t clearbits, uint32_t setbits);
+void arm_boot(void);
+uint32_t *arm_decodeirq(uint32_t *regs);
+void arm_fullcontextrestore(uint32_t *restoreregs) noreturn_function;
+void arm_switchcontext(uint32_t **saveregs, uint32_t *restoreregs);
+void arm_sigdeliver(void);
+uint32_t *arm_doirq(int irq, uint32_t *regs);
+uint32_t *arm_dataabort(uint32_t *regs, uint32_t dfar, uint32_t dfsr);
```

Comment:
It make increase code in up_internal.h

up_XXX and arm_XXX both exist.
I think, we need to discuss about this.

1. keep arm_xxx and up_xxx, and remove common/internal.h and seperate per armvX-X
2. unify both into one, only in arch/arm.

Reply:
True. We need to discuss this. 
But let us merge this PR first, as it is required to verify many features on 8730.
We can take up this activity separately.